client/src/logo.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```

Comment:
logos/images should be saved in assets under the public folder (public/assets_

Reply:
I have removed that.